astro/src/content/docs/lifecycle/manage-users/search/user-search-with-elasticsearch.mdx

```diff
@@ -167,6 +168,12 @@ Finally, below, the `email` field matches the string `dinesh` and the `verified`
 
 Learn more about making [queryString queries in the Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/query-dsl-query-string-query.html).
 
+<AvailableSince since="1.99.9"/>
+Below, only the `phoneNumber` field is searched for phone numbers that start with the `303` area code, and any country code, because `phoneNumber:` is specified.
+
+<JSON title="Example Request JSON For Searching By Phone Number With queryString"
```

Comment:
Felt like too much `RemoteCode` was being used in this area (do we really need another repo for this?) so I chose not to follow that. Changing all of that now seems out of scope. The downside is the `JSON` component looks slightly different.